docker-compose.yml

```diff
@@ -20,13 +20,9 @@ services:
       TS_PROPERTIES: |-
         engine.workers=2
         instance.name=v7.7
```

Comment:
I'm biased about this, it's not overriding single properties, but ALL of them. So it's either teamscale.properties or TS_PROPERTIES. 

Right now we have both which might confuse users. Right now it's even worse as the DB is memory by default (AFAIK), so with that setting we're not persisting data.

Suggestion:
- Either remove  teamscale.properties. If so, I'd opt to also move the license file to docker-compose using TS_LICENSE
- ...or comment out this section as done below.

Reply:
Ah good to know. Given the text that was in the file (overwriting...) I thought that it's only selectively overwriting single settings. 

I changed it so this is hopefully clearer now. I'm not so sure about the license file. IMO copying the license file to the config folder is rather easy and nothing much can go wrong. Copy-pasting the contents of the license file into YAML and then indenting it properly is much more error-prone IMO.

Also, on the file-system you can use symlinks or mount the same config folder into all instances to share the same license. With YAML we'd have to use anchors which are an advanced feature that many users won't grok (I had to google and trial-and-error to get it right as well) and that will be another source of config mistakes and wrecked installations ;-)

Reply:
I also clarified for the JVM opts that this also overrides the ones from jvm.properties. Hope that's correct.

Reply:
Nope that will not override these... Things are complicated, you cannot use TEAMSCALE_OPTS in jvm.properties :P